pgwatch2/metrics/wal_stats/14/metric.sql

```diff
@@ -2,7 +2,7 @@ select
     (extract(epoch from now()) * 1e9)::int8 as epoch_ns,
     wal_records,
     wal_fpi,
-    (wal_bytes / 1000)::int8 as wal_bytes_kb,
+    wal_bytes,
```

Comment:
Thanks for the quick fix, but I think you forgot something that you mentioned in another comment...
For some reason, the wal_bytes are of data type numeric, so a cast to bigint or similar might indeed be necessary?

https://github.com/postgres/postgres/blob/1bc8e7b0991c1eae5fa6dc2d29bb2280efb52740/src/backend/utils/adt/pgstatfuncs.c#L1822-L1823

Reply:
Ah crap, missed that it was numeric, good that someone is paying attention :) ...so indeed core devs considered an overflow possibility..so thatswhy I probably did the division thing in the 1st place...I guess to be on the safe side let's divide by 1024 then? Maybe some data warehouse is writing a petabyte per day indeed - then they have "only" 25 years with byte units :)

Reply:
Well, the time counters in pg_stat_database would overflow after 29 years as well...
I guess the assumption is that users will run through at least one statistics reset before that anyway?

I don't feel confident making a decision here. I guess whoever constructs the dashboards to display this data should look up the units anyway, and then the name of the field `wal_bytes_kb` would indicate what kind of value they are getting anyway.
So if you think that converting to kilobytes is a good idea then I am fine with it.

Reply:
Yes I think kB units are just fine like that when they're explicit -  the metric is not part of default configs / dashboards anyways and better be as future proof as possible - this metrics will stay valid for all future PG versions as well if no breaking catalog changes to pg_stat_wal